src/mappings/helpers.ts

```diff
@@ -577,3 +582,78 @@ function createGraphAccountName(
 export function joinID(pieces: Array<string>): string {
   return pieces.join('-')
 }
+
+function min(a: BigDecimal, b: BigDecimal): BigDecimal {
+  return a < b ? a : b
+}
+
+function max(a: BigDecimal, b: BigDecimal): BigDecimal {
+  return a > b ? a : b
+}
+
+export function calculateOwnStakeRatio(indexer: Indexer): BigDecimal {
```

Comment:
Umm weird, I thought it was supported. I'm using Atom and it doesn't highlight the same errors, probably related to a missing syntax highlighter plugin though.

I'll swap the formulas to use the functions instead of the operators.

Reply:
We decided to leave it as is for now, the operators can be used. It might just be my editor is being too picky. 

The goal is next week to make the whole repository use the operators, but i want to get my editor working first